File: kakao/2018/kakao2018_1.py
def solution(n, arr1, arr2):
    answer=[]
    for i in map(lambda x:x[2:].zfill(n),map(bin,map(lambda x,y:x|y,arr1,arr2))):answer+=[i.replace('0',' ').replace('1','#')]
    return answer

File: kakao/2018/kakao2018_1.cpp
#include <string>
#include <vector>

using namespace std;

vector<string> solution(int n, vector<int> arr1, vector<int> arr2)
{
    vector<string> answer;
    int ret[16],i,j;
    for(i=0;i<n;i++)
	{
        string tmp;
		ret[i]=arr1[i]|arr2[i];
		for(j=0;j<n;j++)
		{
			int target=(1<<(n-1));
			tmp+=(ret[i]&(target>>j))>0?'#':' ';
		}
        answer.push_back(tmp);
	}
    return answer;
}